install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -36,7 +36,7 @@ spec:
             severity: warning
           annotations:
             summary: "Triggers when nodes in a pool have overlapping labels such as master, worker, and a custom label therefore a choice must be made as to which is honored."
-            description: "Node {{ $labels.node }} has triggered a pool alert due to a label change. For more details:  oc logs -f -n {{ $labels.namespace }} {{ $labels.pod }} -c machine-config-daemon"
+            description: "Node {{ $labels.exported_node }} has triggered a pool alert due to a label change. For more details check MachineConfigController pod logs: oc logs -f -n {{ $labels.namespace }} machine-config-controller-xxxxx -c machine-config-controller"
```

Comment:
looks good to me overall, quick question what is node vs exported_node here do?

Reply:
If memory serves correctly: We declare a `node` vector on the metric (see: https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/common/metrics.go#L28-L32C23). When the alert is emitted, the node name in question is passed to the metrics code. However, `$labels.node` refers to the node that the NodeController pod is running on, which may or may not be the node we want to report about. Instead, to explicitly access the `node` vector, we must use `$labels.exported_node` instead.

Reply:
gotcha, thanks!